client/client-lib/src/wallet/mod.rs

```diff
@@ -24,6 +24,13 @@ pub struct WalletClient<'c> {
 }
 
 impl<'c> WalletClient<'c> {
+    /// Returns a bitcoin-address forged from a random tweak and the federations peg-in-descriptor
+    ///
+    /// This function will create a random secret using [KeyPair][bitcoin::KeyPair] which is used to tweak the
```

Comment:
"… create a public/secret keypair. The public key is used to tweak …". Also add that the secret key can be used to prove that one did the peg-in to claim it.